magento/magento2
# [MFTF] Test scenario for "Apply shopping cart rule to a single bundle item" (#28921)

Test scenario for "Apply shopping cart rule to a single bundle item" (#28921)





### Description (*)
Test scenario for the following bug:

1. Create a bundle product with two option each contain two product in it
2. Create one option as main as the title with two simple product (a&b) - (Dropdown)
3. Create the second option as a sub as the title with two simple product(c&d)- (Checkbox)
4. Create a shopping cart rule
5. Select a "percent of product price discount"
6. Add 10 % to the discount amount
7. We should add the main option two product in the rule like( SKU is one of (a&b)
8. Apply the rule
9. Go to the frontend
10. Select one SKU from the dropdown of the main simple product and checkbox one sub simple product
11. Add to cart
12. Go to cart page
13. Check the discount is applied total of two product

### Related Pull Requests


### Fixed Issues (if relevant)

MFTF test for this bug
1. Fixes magento/magento2#28921 

### Manual testing scenarios (*)

1. ...
2. ...

### Questions or comments


### Contribution checklist (*)
 - [*] Pull request has a meaningful description of its purpose
 - [*] All commits are accompanied by meaningful commit messages
 - [*] All new or changed code is covered with unit/integration tests (if applicable)
 - [*] All automated tests passed successfully (all builds are green)
